examples/basic-cdk/package.json

```diff
@@ -0,0 +1,8 @@
+{
+  "name": "compendium",
+  "version": "0.1.0",
+  "dependencies": {
+    "@types/node": "^18.11.18",
+    "compendium-cdk": "file:../.."
```

Comment:
Will this be updated to some released version? Or is this like this on purpose

Reply:
fix via npm workspaces